table/queries/merged_reduced_scans.sql

```diff
@@ -159,7 +159,7 @@ WITH AllScans AS (
 
         ClassifyError(error, source, success, blockpage, page_signature, ExtractServerHeader(received_headers)) AS outcome,
         CONCAT("AS", asn, IF(organization IS NOT NULL, CONCAT(" - ", organization), "")) AS subnetwork,
-        category,
+        IF(category IS NULL, "Uncategorized", category) AS category,
```

Comment:
```suggestion
        IFNULL(category, "Uncategorized") AS category,
```

See https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#ifnull